### ISSUE TITLE
Error during compiling

### ISSUE THREAD

**Issue by a-chacon** [samphippen/rubyfmt#340]
* Ruby version: 3.0.2
* Rubyfmt git sha: I downloaded the tag version 0.7.6 and clone the trunk branch and was almost the same error.
* Os: Debian 11
* Cargo and Rust: 📦 v0.6.8-pre via 🦀 v1.61.0

### Output for the clone of trunk version:
```bash
...
   Compiling rubyfmt v0.6.8-pre (/home/andres/Documentos/Proyectos/rubyfmt/librubyfmt)
   Compiling unicode-normalization v0.1.19
   Compiling addr2line v0.12.2
error: failed to run custom build command for `rubyfmt v0.6.8-pre (/home/andres/Documentos/Proyectos/rubyfmt/librubyfmt)`

Caused by:
  process didn't exit successfully: `/home/andres/Documentos/Proyectos/rubyfmt/target/release/build/rubyfmt-84c65605c74b8343/build-script-build` (exit status: 1)
  --- stderr
  Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:22: release] Error 101

```

### Output for the tag 0.7.6 version:
```bash
...
   Compiling rubyfmt v0.6.8-pre (/home/andres/Documentos/Proyectos/rubyfmt-0.7.6/librubyfmt)
   Compiling unicode-normalization v0.1.19
   Compiling addr2line v0.12.2
error: failed to run custom build command for `rubyfmt v0.6.8-pre (/home/andres/Documentos/Proyectos/rubyfmt-0.7.6/librubyfmt)`

Caused by:
  process didn't exit successfully: `/home/andres/Documentos/Proyectos/rubyfmt-0.7.6/target/release/build/rubyfmt-84c65605c74b8343/build-script-build` (exit status: 1)
  --- stderr
  fatal: no es un repositorio git (ni ninguno de los directorios superiores): .git
  Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:22: release] Error 101

```

What is not found? What I am skipping?

**Comment by gorsing** [comment]
Hi everyone
the error is also reproduced on
```
...
[admin@cold-wood-vs rubyfmt]$ make all
bash -c "find target/release | grep -i 'librubyfmt-.*\.a' | xargs rm; exit 0"
rm: missing operand
Try 'rm --help' for more information.
cargo build --release
   Compiling rubyfmt v0.6.8-pre (/home/admin/rubyfmt/librubyfmt)
   Compiling ureq v2.1.1
error: failed to run custom build command for `rubyfmt v0.6.8-pre (/home/admin/rubyfmt/librubyfmt)`

Caused by:
  process didn't exit successfully: `/home/admin/rubyfmt/target/release/build/rubyfmt-0cebd5e6ffabadaa/build-script-build` (exit status: 1)
  --- stderr
  Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:22: release] Error 101
[admin@cold-wood-vs rubyfmt]$ 
```

    Ruby version: ruby 2.7.3p183 (2021-04-05 revision 6847ee089d) [x86_64-linux]
    Os: Fedora release 32 (Thirty Two)
    Cargo and Rust: package v0.6.8-pre via crab v1.62.0

**Comment by gorsing** [comment]
I upgrade OS to fedora 35. But I think this doesn't meter.
I install rvm 
[admin@localhost ~]$ rvm install 2.7
[admin@localhost ~]$ ruby -v
ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-linux]
```
error: field `spaces` is never read
  --> librubyfmt/src/render_targets.rs:61:5
   |
60 | pub struct BreakableEntry {
   |            -------------- field in this struct
61 |     spaces: ColNumber,
   |     ^^^^^^
   |
note: the lint level is defined here
  --> librubyfmt/src/lib.rs:1:9
   |
1  | #![deny(warnings, missing_copy_implementations)]
   |         ^^^^^^^^
   = note: `#[deny(dead_code)]` implied by `#[deny(warnings)]`
   = note: `BreakableEntry` has derived impls for the traits `Clone` and `Debug`, but these are intentionally ignored during dead code analysis

error: could not compile `rubyfmt` due to previous error
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:22: release] Error 101
[admin@localhost rubyfmt]$ vim librubyfmt/src/render_targets.rs
[admin@localhost rubyfmt]$ vim librubyfmt/src/render_targets.rs
[admin@localhost rubyfmt]$ make all
```

**Comment by btrd** [comment]
I think this PR https://github.com/penelopezone/rubyfmt/pull/343 should help fix things, it did on my end at least.

**Comment by kupkovski** [comment]
I'm having the same error running on v.0.7.6:
```
...
   Compiling quote v1.0.7
   Compiling time v0.1.43
   Compiling dirs-sys v0.3.6
   Compiling filetime v0.2.14
error: failed to run custom build command for `rubyfmt v0.6.8-pre (/home/andre/rubyfmt/librubyfmt)`

Caused by:
  process didn't exit successfully: `/home/andre/rubyfmt/target/release/build/rubyfmt-10e29e57e70ed59b/build-script-build` (exit status: 1)
  --- stderr
  Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:22: release] Error 101

```

I'm running it on an `Ubuntu 22.04.1 LTS` over a `Linux 5.15.0-52-generic x86_64`

**Comment by reese** [comment]
@kupkovski  0.7.6 is an old version, this should be fixed on 0.8.0. Does updating the version work for you?
